data/presets/presets/highway/motorway_link.json

```diff
@@ -4,7 +4,7 @@
         "name",
         "ref_road_number",
         "oneway",
-        "maxspeed",
+        "maxspeed/advisory",
```

Comment:
`maxspeed=*` is a valid tag on `higway=motorway_link` in some countries and should not be removed.